diff --git a/third-party/intel-hexl/intel-hexl.cmake b/third-party/intel-hexl/intel-hexl.cmake
index 5bdb3d3..a79a5ed 100644
--- a/third-party/intel-hexl/intel-hexl.cmake
+++ b/third-party/intel-hexl/intel-hexl.cmake
@@ -7,7 +7,7 @@ option(INTEL_HEXL_PREBUILT OFF) # Set to ON/OFF to use prebuilt installation
 message(STATUS "INTEL_HEXL_PREBUILT ${INTEL_HEXL_PREBUILT}")

 if (INTEL_HEXL_PREBUILT)  # Skip download from gitlab
-  find_package(HEXL 1.2.1
+  find_package(HEXL 1.2.3
         HINTS ${INTEL_HEXL_HINT_DIR}
         REQUIRED)
   if (NOT TARGET HEXL::hexl)
@@ -31,7 +31,7 @@ else()
     # Download and install Intel HEXL ...
     # ------------------------------------------------------------------------------
     set(INTEL_HEXL_GIT_REPO_URL https://github.com/intel/hexl)
-    set(INTEL_HEXL_GIT_LABEL 1.2.1)
+    set(INTEL_HEXL_GIT_LABEL 1.2.3)

     set(INTEL_HEXL_DEBUG OFF) # Set to ON/OFF to toggle debugging
     set(INTEL_HEXL_SHARED_LIB ${BUILD_SHARED}) # Set to ON/OFF to toggle shared library build
